lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -69,3 +69,7 @@
         padding: 0;
     }
 }
+
+.mat-list-base .mat-list-item .mat-line {
```

Comment:
what do exactly mean with longer? this css seems to be changing .mat-list-item globally are we sure we don't have unexpected impacts on the apps?

Reply:
those styles should be scoped to the notification history, not being global ones, also as @mauriziovitale said, what does this change do? can you add some screenshots or explanation?

Reply:
scoped the change to notification history. Screenshots for your reference are present https://alfresco.atlassian.net/browse/ACS-4277.